docker-compose.yaml

```diff
@@ -53,8 +53,3 @@ volumes:
   database_data:
     driver: local
   matplotlib-config:
-    emptyDir: {}
-
-volumeMounts:
-  - name: matplotlib-config
-    mountPath: "${MPLCONFIGDIR}"
```

Comment:
I think this should be written as 

```
volumes:
  - ${MPLCONFIGDIR}:/something:z
```

but I wasn't sure to which container it should be mounted and to what path.

Reply:
I have added a new commit with the mapping. Please take a look at it.